ci/build.sh

```diff
@@ -17,17 +16,22 @@ trap 'err_report $LINENO' ERR
 if [ -n "$GITHUB_TOKEN" ]; then
   git config --global user.email "travis@travis-ci.org"
   git config --global user.name "Travis CI"
-  git remote add origin-repo https://${GITHUB_TOKEN}@github.com/SumoLogic/sumologic-kubernetes-collection.git > /dev/null 2>&1
+  git remote add origin-repo "https://${GITHUB_TOKEN}@github.com/SumoLogic/sumologic-kubernetes-collection.git" > /dev/null 2>&1
   git fetch --unshallow origin-repo
-  git checkout $TRAVIS_PULL_REQUEST_BRANCH
+  git checkout "$TRAVIS_PULL_REQUEST_BRANCH"
```

Comment:
```suggestion
  git checkout "${TRAVIS_PULL_REQUEST_BRANCH}"
```

? 🤔

Reply:
Many variables in this script are used without {} so this should be fixed everywhere actually, not only in here.

Reply:
Adressed in https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/919